cmd/pktvisord/CrashpadHandler.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#define NOMINMAX
+
+#include "visor_config.h"
+#include <client/crash_report_database.h>
+#include <client/crashpad_client.h>
+#include <client/settings.h>
+
+namespace crashpad {
+
+static bool start_crashpad_handler(base::FilePath::StringType token, base::FilePath::StringType url, base::FilePath::StringType handler_path)
+{
+    std::map<std::string, std::string> annotations;
+    std::vector<std::string> arguments;
+    CrashpadClient client;
+    bool rc;
+
+    annotations["format"] = "minidump";
+    annotations["product"] = "pktvisor";
+    annotations["database"] = "pktvisor";
+    annotations["version"] = VISOR_VERSION_NUM;
+    annotations["token"] = token;
+    base::FilePath::StringType db_path("crashpad");
```

Comment:
is this a file on disk? i see "no-rate-limit" below, but we would need to be careful if this writes to disk, we can't let the disk fill

Reply:
the rate limit should be a dump per hour